src/frontend/magnify/src/views/auth/register/AuthRegisterView.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { AuthRegisterView } from './index';
+
+export default {
+  title: 'Views/Auth/AuthRegisterView',
+  component: AuthRegisterView,
+} as ComponentMeta<typeof AuthRegisterView>;
+
+const Template: ComponentStory<typeof AuthRegisterView> = (args) => {
+  return (
+    <RouterProvider
```

Comment:
ditto

Reply:
Yes for the moment the package is too linked to the router. Gotta undo it